src/test/java/emissary/core/sentinel/protocols/rules/AllMaxTimeTest.java

```diff
@@ -51,4 +60,54 @@ void notOverTimeLimit() {
         Rule rule = new AllMaxTime("rule1", "TestPlace", 2, 0.75);
         assertFalse(rule.overTimeLimit(placeAgentStats));
     }
+
+    @Test
+    void condition() {
+
+        Protocol.PlaceAgentStats lowerStats = new Protocol.PlaceAgentStats("ToLowerPlace");
+        lowerStats.update(5);
+        lowerStats.update(6);
+        lowerStats.update(9);
+
+        Protocol.PlaceAgentStats upperStats = new Protocol.PlaceAgentStats("ToUpperPlace");
+        upperStats.update(5);
+        upperStats.update(7);
```

Comment:
When reading the assertions, it's hard to track the correlation between these magic numbers and the values used in the rule constructors.  Might be easier to follow if we added named values and used those when resetting the pool size or instantiating rules. If I'm having a hard time following while this code is new, I can imagine it being much harder for a new dev to follow it in the future.
```suggestion
        final int DEFAULT_TIME_LIMIT = 5;
        Protocol.PlaceAgentStats lowerStats = new Protocol.PlaceAgentStats("ToLowerPlace");
        lowerStats.update(DEFAULT_TIME_LIMIT);       // agent A
        lowerStats.update(DEFAULT_TIME_LIMIT + 3);   // agent B
        lowerStats.update(DEFAULT_TIME_LIMIT + 1);   // agent C

        Protocol.PlaceAgentStats upperStats = new Protocol.PlaceAgentStats("ToUpperPlace");
        upperStats.update(DEFAULT_TIME_LIMIT);     // agent X
        upperStats.update(DEFAULT_TIME_LIMIT + 2);  // agent Y
        
        final int TOTAL_BUSY_AGENTS = lowerStats.getCount() + upperStats.getCount();
```
```java        
        ...
        setPoolSize(pool, TOTAL_BUSY_AGENTS + 1);
        Rule rule = new AllMaxTime("rule1", "To(?:Lower|Upper)Place", DEFAULT_TIME_LIMIT, 1.0);
```